### DISCUSSION TITLE
Can we get a new release soon ?

### DISCUSSION THREAD

**Discussion by Roobxyz** [ankitpokhrel/jira-cli#683] [General]
question is title

**Comment by ankitpokhrel** [comment]
Hey @Roobxyz I am working on a new release, will possibly make a release in a few weeks. 

In the mean time, you can either [compile from source](https://github.com/ankitpokhrel/jira-cli#development) or if you are using homebrew you can build binary from `HEAD` (requires make, git and go).

```sh
brew uninstall jira-cli
brew install jira-cli --HEAD
```

[Docker build](https://github.com/ankitpokhrel/jira-cli/pkgs/container/jira-cli/164306155?tag=main) against `main` branch is also available.
